### ISSUE TITLE
Populate DB with Data

### ISSUE THREAD

**Issue by K-shamG** [sysc4806/SYSC4806-WDWH#22]
Issue #15 and #17 must be done first so the DB is created properly

**Comment by K-shamG** [comment]
I think we just have to create a data.sql script for this. 
https://www.baeldung.com/spring-boot-data-sql-and-schema-sql 
https://stackoverflow.com/questions/38040572/spring-boot-loading-initial-data

**Comment by andrewdlqnguyen** [comment]
This is what i made for my data.sql. Not much but from what i found online. You probably had something similar or better.

This one is if you want to make a schema.h2.sql but we'd have to configure the hibernate to not auto generate from our classes.
`CREATE TABLE  'course' (
  [id] INTEGER NOT NULL AUTO_INCREMENT,
  [year] INTEGER NOT NULL,
  [name] VARCHAR(128) NOT NULL,
  [code] VARCHAR(128) NOT NULL,
  PRIMARY KEY (id)
);`

This other one i found is just making a data.sql. I started with something like this.
`-- INSERT INTO courses (year, name, code) VALUES
-- (4, 'Software Engineering Lab', 'SYSC 4806'),
-- (4, 'Computer Systems Architecture', 'SYSC 4507'),
-- (4, 'Software Validation', 'SYSC 4101'),
-- (3, 'Operating Systems', 'SYSC 4001'),
-- (3, 'Software Development Project', 'SYSC 3110'),
-- (3, 'Real-Time Concurrent Systems', 'SYSC 3303');

-- INSERT INTO categories (name, learningoutcomes) VALUES
--
-- INSERT INTO learningoutcomes (name, category) VALUES
--
-- INSERT INTO programs (name, )`

**Comment by andrewdlqnguyen** [comment]
oops

**Comment by K-shamG** [comment]
This is not done via data.sql or PostgreSQL, but it is working and mapping properly through Application so I'm going to consider this closed. We can open a new issue if we want to persist the data with PostgreSQL later.
